setup.cfg

```diff
@@ -30,6 +30,7 @@ exclude =
     __pycache__/,
     node_modules/,
     */migrations/
+    */vendored/
```

Comment:
Why not simply fork their repository and include it as [git dependency in poetry](https://python-poetry.org/docs/dependency-specification/#git-dependencies) (with full hash, not partial hash like they suggest in docs) or a sub-module? It would make it much easier to review and see diffs in the future as well

Reply:
I don't want us to maintain a fork or create an illusion that we do.